cyral/resource_cyral_sidecar.go

```diff
@@ -94,7 +96,18 @@ func resourceSidecar() *schema.Resource {
 				),
 			},
 			"log_integration_id": {
-				Description: "ID of the log integration mapped to this sidecar.",
+				Description: "ID of the log integration mapped to this sidecar, used for Cyral activity logs.",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Deprecated:  "Since sidecar v4.8. Use `activity_log_integration_id` instead.",
```

Comment:
```suggestion
				Deprecated:  "Since sidecar v4.8. Use `activity_log_integration_id` instead.",
				ConflictsWith: []string{"activity_log_integration_id"}
```